cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -1261,6 +1298,22 @@ describe('CourseOutlinePage', function() {
             expect($('.edit-show-correctness').length).toBe(1);
         });
 
+        it('can show correct editors for self_paced course with custom pacing', function (){
+            var mockCourseJSON = createMockCourseJSON({}, [
+                createMockSectionJSON({}, [
+                    createMockSubsectionJSON({}, [])
+                ])
+            ]);
+            createCourseOutlinePage(this, mockCourseJSON, false);
```

Comment:
It seemed like `mockCourseJSON` is reassigned for specific scenarios, but in this case I don't think I need to reassign it since it is the same as the previous assignment statement!